apps/nextjs-website/react-components/components/Accordion/Accordion.tsx

```diff
@@ -14,19 +14,13 @@ const Accordion = (props: AccordionProps) => {
     description,
     accordionItems,
     theme,
-    layout = 'left',
+    layout = 'center',
+    textAlignment = layout === 'center' ? 'center' : 'left',
```

Comment:
```suggestion
    layout,
    textAlignment,
```
No need for default value when fields are required.